tests/fixture/bootstrapmonitor/wait.go

```diff
@@ -148,9 +149,9 @@ func WaitForCompletion(
 	return nil
 }
 
-// Determines the current disk usage for the specified directory
-func getDiskUsage(log logging.Logger, dir string) string {
-	cmd := exec.Command("du", "-sh", dir)
+// Determines the current disk usage (in bytes) for the specified directory
+func getDiskUsage(log logging.Logger, dir string) uint64 {
+	cmd := exec.Command("du", "-sb", dir)
```

Comment:
unrelated to this PR, but, perhaps it's better to use an[ API call ](https://www.includehelp.com/golang/get-the-disk-usage-information.aspx)instead of using `du`?